nrcatalogtools/lvc.py

```diff
@@ -789,41 +806,42 @@ def get_nr_to_lal_rotation_angles(
         ]
 
         # Check if time series data of required reference data is present
-        RefCheckInterpReq, AbsentInterpAttrs = CheckNRAttrs(H5File, ReqTSAttrs)
+        ref_check_interp_req, absent_interp_attrs = check_nr_attrs(
+            h5_file, req_ts_attrs
+        )
 
-        if RefCheckInterpReq is False:
+        if ref_check_interp_req is False:
             raise Exception(
-                "Insufficient information to compute the LAL source frame at given reference time.",
-                f"Missing information is {AbsentInterpAttrs}.",
+                f"Insufficient information to compute the LAL source frame at given reference time. Missing information is {absent_interp_attrs}."
```

Comment:
Ensure line length is less than 120 by breaking the string up.

Reply:
I suggest keeping the original format

```
                "Insufficient information to compute the LAL source frame at given reference time.",
                f"Missing information is {absent_interp_attrs}."
```

Reply:
nvm, these have alreayd been fixed